lib/NAnt/NAnt.exe.config

```diff
@@ -2497,6 +2497,166 @@
                         </task>
                     </tasks>
                 </framework>
+                <framework
+                    name="mono-4.5"
```

Comment:
Am I seeing two 4.5s?

Reply:
Yes, there might be a better way to handle it, but it works. It looks kind of weird since mono-4.0 mostly got merged to 4.5, so it partially looks like it 4.5 twice.

This was originally different but in your previous review, you said to add another framework rather than replace the old mono-4.0 one. 
>Typically in this file you simply add ANOTHER framework, so the old one is still here.